### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by m-cat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ skynet-cli is a lightweight CLI to interact with [Skynet](https://siasky.net).
 Skynet is the decentralized CDN and file sharing platform for devs and the
 storage foundation for a Free Internet!
 
+## Installing
+
+If you have [Go](https://golang.org/cmd/go/) installed you can run:
```
Comment (MSevey) [inline README.md:12]: that gets the repo, don't they still need to run `make release` to install the binaries?

File: cmd/skynet/main.go [hunk 2]
```diff
@@ -12,11 +12,21 @@ import (
 )
 
 const (
-	// mainDocFile is the filepath to the top level doc file created by cobra
-	mainDocFile = "./doc/skynet.md"
+	// binDescription is the binary description.
+	binDescription = `Perform actions related to Skynet, a file sharing and data publication platform
+on top of Sia.`
```
Comment (MSevey) [inline cmd/skynet/main.go:17]: nit: `built on top of Sia`

File: README.md (hunk 1 above)
Comment (m-cat) [reply README.md:12]: Don't think so, i.e. look at the way we install `analyze`:

```
go get gitlab.com/NebulousLabs/analyze
```

Comment (MSevey) [reply README.md:12]:
```
matt@Carl:~/go/src/github.com/NebulousLabs$ go get github.com/NebulousLabs/skynet-cli
package github.com/NebulousLabs/skynet-cli: no Go files in /home/matt/go/src/github.com/NebulousLabs/skynet-cli
```
 `go get` tries to install packages for a projects. I think you need the `-u` and the `/...` to install a new project

```
matt@Carl:~/go/src/github.com/NebulousLabs$ go get -u github.com/NebulousLabs/skynet-cli/...
matt@Carl:~/go/src/github.com/NebulousLabs$ skynet
Use Skynet to upload and share content

Default Skynet Portal: https://siasky.net 

```

**Pull request merged**